src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write your code here
+const population = document.querySelectorAll('.population');
+const totalPopulation = document.querySelector('.total-population');
+const averagePopulation = document.querySelector('.average-population');
+let total = 0;
+
+population.forEach(item => {
+  const numberPopulation = +(item.innerText.split(',').join(''));
+
+  total += numberPopulation;
+});
+
+const averageNumberPop = total / population.length;
+
+function thousandSeparator(num) {
```

Comment:
Let's get used to name functions according to these rules
![image](https://user-images.githubusercontent.com/96409563/209646191-daf8795e-220d-40f3-8848-24623ccda079.png)